public/content/public-goods-funding/index.md

```diff
@@ -0,0 +1,307 @@
+
+3. **Negative externalities**: Another downside with democratic funding schemes is the cost borne by the minority. In democratic systems, those who may not necessarily need a public good are forced to pay the same as those who need it the most. Consider the case where Bob (an introvert who prefers staying indoors) is forced to pay taxes to subsidize the creation of public parks for people like Alice (an extrovert who enjoys staying outdoors), even though Bob himself might never get to use the park. This is a trivial example, but there are other real-world situations where the minority in society are (unfairly) required to pay for public goods on behalf of others who need the public good but aren't willing or able to pay for it.  
+
+#### Plutocratic voting {#plutocratic-voting}
+
+Some funding models use a plutocratic mechanism in which an individual's financial power determines their influence on the process of providing public goods. These systems are typically described as "one dollar, one vote". In theory, plutocratic funding is ideal since it allows for expressing unequal preference for public goods—but it also has problems:
+
+1. **Serving the rich**: Plutocratic voting prioritizes the preferences of the wealthy (who are often the minority) to the detriment of average income-earners that comprise the majority. Thus, public goods that matter to the majority (and have bigger societal benefits) may never receive funding.
+
+2. **Cost burdens**: It is possible for middle-class citizens to leverage crowdfunding mechanisms in order to compete against wealthy indivdiduals and collectively express their preferences for a public good. But this is considerably unfair since they still have to bear unfavorably higher costs when expressing preferences.
+
+3. **Negative externalities**: Public goods created via plutocratic mechanisms often produce negative externalities, mostly because of the need to maximize returns on investment. A classic example is the Internet as it exists today: because Internet businesses (e.g., social media platforms) cannot charge consumers directly, they rely on advertising to recoup investment in infrastructure. This is even as reports of the [negative effects of targeted advertising on individuals](https://www.vox.com/platform/amp/the-goods/2020/4/9/21204425/targeted-ads-fertility-eating-disorder-coronavirus) continue to surface.
+
+## Improving public goods funding with blockchain technology {#blockchain-for-public-goods-funding}
+
+The problems explained in the previous section lead to "coordination failures" that stall the development of public goods. However, many believe the existence of decentralized blockchain networks, such as Ethereum, can help solve the problem of efficiently funding and developing public goods.
```

Comment:
i dont think this is important here and should be removed from this page

Reply:
Hi @konopkja. I think it helps to show why a reader should see quadratic funding (or any crypto-based model of funding public goods) as an alternative to the more common approaches (e.g., government funding, private funding via venture capital, and popular crowdfunding schemes like KickStarter). The other part already addresses the incentive problems with some of these models, but the other section highlights more concrete ways in which incentives may be misaligned. 

I do agree the text could be shorter and more direct to the point, which is why I rewrote most of the section and removed redundant parts. Let me know if this addresses your concerns.